src/components/UserInfo.vue

```diff
@@ -1,24 +1,42 @@
+<!--suppress XmlInvalidId -->
 <template>
-  <div id="user-info">
-    <v-alert :value="!!successMessage" type="success" dismissible>{{ successMessage }}</v-alert>
-    <v-alert :value="!!errorMessage" type="error" dismissible>{{ errorMessage }}</v-alert>
+  <div id="user-info" v-if="dataReady">
+    <v-alert v-model="successState" type="success" dismissible>{{ successMessage }}</v-alert>
+    <v-alert v-model="errorState" type="error" dismissible>{{ errorMessage }}</v-alert>
 
     <h1>Update - {{ user.username }}</h1>
-    <v-row no-gutters>
-      <v-col class="col-7">
-        <label for="user-name" class="disabled required">User Name</label>
-        <v-text-field dense outlined disabled id="user-name" v-model="user.username"
-        required :rules="[v => !!v || 'Username is required']"/>
-
-        <label for="first-name" class="required">First Name</label>
-        <v-text-field dense outlined id="first-name" v-model="user.firstName" 
-        required :rules="[v => !!v || 'First Name is required']"/>
-
-        <label for="last-name" class="required">Last Name</label>
-        <v-text-field dense outlined id="last-name" v-model="user.lastName"
-        required :rules="[v => !!v || 'Last Name is required']"/>
-      </v-col>
-    </v-row>
+    <v-card outlined class="subgroup">
+      <h2>User Details</h2>
```

Comment:
I put a `v-card` around this just to get the spacing right. Without `v-card` the "Save" button didn't have any room around it. Please go ahead and change this if you have opinions.

BEFORE
![image](https://user-images.githubusercontent.com/1767127/84467876-38d44880-ac32-11ea-95b4-619ff491f243.png)

AFTER
![image](https://user-images.githubusercontent.com/1767127/84467898-4ab5eb80-ac32-11ea-8758-610e72791c72.png)

Reply:
I think it looks better with the card since we have separate save buttons